.github/workflows/release.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         # note that sdl-schema.json should be available in the location below from the previous maven build.
         mv sdl-lang/target/classes/sdl-schema.json static/json-schema-viewer/schemas/sdl-schema-${MVN_RELEASE_VERSION}.json
         git add static/json-schema-viewer/schemas/sdl-schema-${MVN_RELEASE_VERSION}.json
+        git rm static/json-schema-viewer/schemas/sdl-schema-${MVN_RELEASE_VERSION}-SNAPSHOT.json
```

Comment:
What happens if the SNAPSHOT doesn't exist? Wouldn't this step fail and abort the whole pipeline? Maybe we should add `--ignore-unmatch` to `git rm` ?

Reply:
Usually it should exist, but yes, better be safe.